### ISSUE TITLE
Feed Screen Base page re-design

### ISSUE THREAD

**Issue by ugur-eren** [keep-starknet-strange/joyboy#101]
### OD Hack

This issue is part of ODHack 4.0:

description - https://x.com/OnlyDust_com/status/1791035111992934563
guidelines - https://onlydust.notion.site/ODHack-Common-Guidelines-Project-leads-5e21836f85ad493f9ca429770af0ad40

Please read the guidelines and make sure you follow them.

**Please write your thoughts and how long you think it will take to complete the issue in the comments.**

### Feed Screen Base re-design

Feed screen should be re-designed to match the new design. The new design is available at Figma: https://www.figma.com/design/NIJ68MV1AeI8y283woXiM0/JoyBoy?node-id=235-5345&t=QIIZCaGcwk8ziRaQ-0

Current Feed screen be found at: https://github.com/keep-starknet-strange/joyboy/blob/main/JoyboyCommunity/src/modules/feed/index.tsx

You should implement the design as close as possible and should match 1:1 except for the post card. The post card should be implemented as a separate component and issue.

In short, you should implement the following:
- Header
- Stories section
- Bottom bar
- Create post button

Please battle test it before doing a PR.

### Communication

https://t.me/JoyboyStarknet/949

May the Joy be with you!

**Comment by kateberryd** [comment]
I would love to work on this

**Comment by josephchimebuka** [comment]
i

**Comment by ugur-eren** [comment]
@josephchimebuka Hi, I've assigned this to @kateberryd since she is the first one and included the required time. However we have other issues to look at
